cfme/infrastructure/config_management.py

```diff
@@ -441,17 +445,26 @@ class MgrAll(CFMENavigateStep):
     prerequisite = NavigateToAttribute('appliance.server', 'LoggedIn')
 
     def step(self):
-        if self.obj.appliance.version > '5.7.0.8':
-            self.prerequisite_view.navigation.select('Configuration', 'Management')
-        else:
+        if self.obj.appliance.version < '5.7':
             self.prerequisite_view.navigation.select('Configuration', 'Configuration Management')
+        if self.obj.appliance.version >= '5.8':
+            self.prerequisite_view.navigation.select('Automation', 'Ansible Tower', 'Explorer')
+        if self.obj.appliance.version > '5.7' and self.obj.appliance.version < '5.8':
```

Comment:
**Optional**
Tempted to say required but I don't think we have anything for this in our coding standards.  I think using `elif` should be a required change here though.
Could you switch the order here so its:

```python
if x < 5.7:
elseif x >= 5.7 and < 5.8
elseif x >= 5.8